frontend/src/components/RecordsList.tsx

```diff
@@ -30,7 +30,7 @@ import {
   defaultColDef,
   MAX_ROWS_EXPORT,
 } from "../shared/helpers";
-import { ErrorMessage, Toolbar } from "../shared/tableElements";
+import { ErrorMessage, Toolbar } from "../shared/components/Toolbar";
```

Comment:
Renamed this file. React component file names should be in PascalCase by traditional standards.